content/fr/docs/concepts/observability-primer.md

```diff
@@ -96,7 +96,7 @@ lorsqu'ils sont mis en corrélation avec une trace ou un span.
 Pour plus d'informations sur les logs et leur rôle dans OpenTelemetry,
 référez-vous à la page [Logs](/docs/concepts/signals/logs/).
 
-### Spans
+### Spans {#spans}
```

Comment:
@iguitton - you don't actually need the ID here since the heading text corresponds to `#spans`, so maybe consider dropping the ID.

Reply:
If you decide to remove it, consider removing it for the `### Logs` heading as well.